coreos-meta-translator/trans.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+
+from argparse import ArgumentParser, FileType
+from datetime import datetime
+import json
+import os
+
+
+def ensureDup(inp, out, inp_key, out_key):
+    if out.get(out_key, None) == None:
+        out[out_key] = inp.get(inp_key)
+        return out
+    if out.get(out_key) != inp.get(inp_key):
+        print("Input Files do not appear to be for the same release")
+        raise SystemExit(1)
+    return out
+
+def url_builder(stream, version, arch, path):
+    return f"https://fcos-builds.s3.amazonaws.com/prod/streams/{stream}/builds/{version}/{arch}/{path}"
+
+def get_extension(path, ext_count):
+    return ".".join(path.split(".")[-ext_count:])
+
+
+parser = ArgumentParser()
+parser.add_argument("--workdir", help="cosa workdir", required=True)
+parser.add_argument("--build-id", help="build id", required=True)
```

Comment:
Optional: could make this optional so we just default to the last build (first build id in `builds.json`).